components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/widget/WidgetConfigs.java

```diff
@@ -1,24 +1,24 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Revert these changes in the license header.
Applies for other places as well.

Reply:
fixed with 910980c413d9aea940c47a751f949593c1fc5ec0